utils/utils.ts

```diff
@@ -51,20 +55,49 @@ export function getEnvironmentRequiredVars() {
   }
 
   const logLevel = process.env.LOG_LEVEL ? process.env.LOG_LEVEL : "info";
+
   const uri = process.env.API_URL ? process.env.API_URL : "ws://127.0.0.1:9944";
+
   const userPassword = process.env.UI_USR_PWD
     ? process.env.UI_USR_PWD
     : "mangata123";
+
   const uiUri = process.env.UI_URL
     ? process.env.UI_URL
     : "https://staging.mangata.finance/";
+
   const mnemonicMetaMask = process.env.MNEMONIC_META
     ? process.env.MNEMONIC_META
     : " oh oh";
+
   const mnemonicPolkadot = process.env.MNEMONIC_POLK
     ? process.env.MNEMONIC_POLK
     : " oh oh";
 
+  const ethereumWsUrl = process.env.ETHEREUM_WEBSOCKET_URL
+    ? process.env.ETH_APP_ADDRESS
+    : "ws://localhost:8545";
+
+  const substrateWsUrl = process.env.SUBSTRATE_WEBSOCKET_URL
+    ? process.env.ETH_APP_ADDRESS
+    : "ws://localhost:9944";
```

Comment:
I think that this guy should match with API_URL, shouldn't it? It should be the target test environment. So maybe we can reuse it from above.